luaui/rml_setup.lua

```diff
@@ -73,12 +73,37 @@ end
 ]]
 
 -- when "cursor: normal" is set via RCSS, "cursornormal" will be sent to the engine... and so on for the rest
-RmlUi.SetMouseCursorAlias("default", 'cursornormal')
```

Comment:
Is there convention around single vs double quotes? if so please set your linter to adhere, because this will get annoying.

Reply:
Unfortunately we don't have a convention. Personally, I favour single quotes for programmatic values (keys, tags, etc.) and double quotes for natural language values (messages, display strings, etc.).